utils/build/docker/nodejs/express/debugger/index.js

```diff
@@ -1,9 +1,67 @@
 'use strict'
 
-const logHandler = require('./log_handler')
+const Pii = require('./pii')
 
 module.exports = {
-  initRoutes (app, tracer) {
-    app.get('/log', logHandler)
+  initRoutes (app) {
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+
+    app.get('/debugger/log', (req, res) => {
+      res.send('Log probe') // This needs to be line 20
+    })
+
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+    // Padding
+
+    app.get('/debugger/pii', (req, res) => {
+      const pii = new Pii() // eslint-disable-line no-unused-vars
+      res.send('Hello World') // This needs to be line 64
```

Comment:
The other languages responds with a string that looks like this (example from the Java code):

```java
String value = pii.TestValue;
String customValue = customPii.TestValue;
"PII " + value + ". CustomPII" + customValue;
```

However, I can't see any place where the response of the `/debugger/pii` endpoint is ever evaluated. That's `TestValue` used for? Why this special response?

Reply:
In .net, when variable is not used, build optimizations can remove it, so the value was used in return. Other libraries aligned with the schema, but it's not necessary.